src/guides/v2.3/javascript-dev-guide/widgets/widget_dialog.md

```diff
@@ -195,6 +195,49 @@ This example uses some CSS classes that are used for minicart.
 </div>
 ```
 
+The following example shows a PHTML file using the script and custom CSS classes that are not styled:
```

Comment:
What does it mean to say "custom CSS classes that are not styled."? Do you mean that the classes have no defined properties?

Reply:
Hi, @dobooth !

> Do you mean that the classes have no defined properties?

Yes. The classes `.dropdown-wrap`, `dropdown-button` are specified but for them @volodymyrDashuk did not provide any styles (for example, `backgroud`, etc).

Thank you!

Reply:
Hi @dobooth 
Yes exactly.
Since the existing documentation uses an [example](https://devdocs.magento.com/guides/v2.4/javascript-dev-guide/widgets/widget_dialog.html#code-sample) using classes that already have properties.
I decided to clarify this in order not to mislead the reader.

Reply:
Excellent. Thanks to you both.